--- eng-lex-sample.train.key	2013-04-11 16:36:18.000000000 +0200
+++ eng-lex-sample.train.fixed.key	2013-04-11 16:36:18.000000000 +0200
@@ -2861,7 +2861,7 @@
 free free.40268 free%5:00:01:unoccupied:00
 free free.40269 free%3:00:00::
 free free.40271 free%3:00:00::
-free free.40272 free_will%1:07:00:: free_will%1:26:00::
+free free.40272 free_will%1:07:00::
 free free.40274 free%5:00:00:unpaid:00
 free free.40275 free%3:00:00::
 free free.40277 free%5:00:00:unpaid:00
@@ -4019,7 +4019,7 @@
 natural natural.40286 natural%5:00:00:normal:01
 natural natural.40287 natural_history%1:09:00:: P
 natural natural.40289 natural%3:00:01::
-natural natural.40290 natural_language_processing%1:10:00::
+natural natural.40290 natural_language_processing%1:09:00::
 natural natural.40293 natural%5:00:00:normal:01 natural%5:00:00:spontaneous:00
 natural natural.40294 natural_science%1:09:00::
 natural natural.40296 natural%3:00:01::
@@ -4296,11 +4296,11 @@
 post post.40153 post%1:15:00:: U
 post post.40156 post%1:06:00::
 post post.40157 post%1:04:00::
-post post.40161 local_post_office%1:14:01:: post_office%1:14:01::
+post post.40161 local_post_office%1:14:00:: post_office%1:14:01::
 post post.40162 post%1:06:00::
 post post.40165 post_office%1:14:01::
 post post.40166 P
-post post.40171 local_post_office%1:14:01:: post_office%1:14:01::
+post post.40171 local_post_office%1:14:00:: post_office%1:14:01::
 post post.40172 post%1:06:00::
 post post.40175 post%1:06:00::
 post post.40177 post%1:10:00::
